docs/host_for_multiuser/setup.md

```diff
@@ -62,90 +64,97 @@ Follow the steps below to setup `multiuser` mode.
+
+### Setup Database
+- Set up your own mysql (or mariadb) server or use docker compose mysql
+- Below are the instructions for using mysql with docker compose.
+
+1. Edit configs.
+    - Edit studio/config/.env
+      - Set `MYSQL_SERVER` to db server host or ip
+        - \*For docker platform, the fixed value `db:3306` is fine.
+      - Set `MYSQL_ROOT_PASSWORD` to {DB_ROOT_PASSWORD}, which you have decided.
+      - Set `MYSQL_DATABASE` to {YOUR_DATABASE_NAME}, which you have decided.
+      - Set `MYSQL_USER` to {DB_USER_NAME}, which you have decided.
+      - Set `MYSQL_PASSWORD` to {DB_USER_PASSWORD}, which you have decided.
+2. Install & run mysql server.
     ```bash
-    mysql -u root -p
-    ```
-3. Create database for your project.
-    ```sql
-    CREATE DATABASE YOUR_DATABASE_NAME;
+    docker compose -f docker-compose.dev.multiuser.yml up db -d
     ```
-4. Create user for your project.
-    ```sql
-    CREATE USER 'DB_USER_NAME'@'localhost' IDENTIFIED BY 'DB_USER_PASSWORD';
-    ```
-5. Grant all privileges to the user for the database.
-    ```sql
-    GRANT ALL PRIVILEGES ON YOUR_DATABASE_NAME.* TO 'DB_USER_NAME'@'localhost';
+    - The database and db_user are automatically generated based on the .env settings.
+3. Connect to mysql server.
+    ```bash
+    docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
     ```
+    - If a connection to the database server is available, the setup was successful.
 
-### Set OptiNiSt config
-1. Edit `studio/config/.env`
+### Setup & Run OptiNiSt
+
+#### For Docker Platform
+
+##### Setup Backend
+- See [OptiNiSt installation guide](../installation/docker_for_developer.md).
+- After building the docker environment, run following commands
+
+###### 1. Set OptiNiSt config
+- Edit `studio/config/.env`
     - Change `SECRET_KEY` to any random string.
     - Change `USE_FIREBASE_TOKEN` to `True`.
     - Change `IS_STANDALONE` to `False`
-    - Set `MYSQL_SERVER` to your host
-    - Set `MYSQL_DATABASE` to {YOUR_DATABASE_NAME}, which you created in the previous step.
-    - Set `MYSQL_USER` to {DB_USER_NAME}, which you created in the previous step.
-    - Set `MYSQL_PASSWORD` to {DB_USER_PASSWORD}, which you created in the previous step.
-    - `MYSQL_ROOT_PASSWORD` can be left commented.
-
-### Setup Frontend
-1. Install node.js(version 20)
-    - https://nodejs.org
-2. Install yarn
-    ```bash
-    npm install -g yarn
-    ```
-3. Install frontend requirements
-    ```bash
-    cd frontend
-    yarn install
-    ```
-4. Build frontend
-    ```bash
-    yarn build
-    ```
 
-### Setup Backend
-- See OptiNiSt installation guide.
+###### 2. Start backend (Database is set up on startup)
+```bash
+docker compose -f docker-compose.dev.multiuser.yml up studio-dev-be -d
+```
+
+###### 3. Insert initial data
+```bash
+docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
```

Comment:
> Is the DB_DOCKER_CONTAINER_NAME here for studio-dev-be or db?

Same as below
- https://github.com/arayabrain/barebone-studio/pull/364#discussion_r1653800596

> Is there a reason the SQL initial values

The procedure is basically described in the following flow chart.

1. basic application setup
2. initial data registration
3. startup confirmation

Initial data registration is 2. because alembic needs to be executed before data registration.